clang/docs/ReleaseNotes.rst

```diff
@@ -192,6 +192,8 @@ Removed Compiler Flags
 Attribute Changes in Clang
 --------------------------
 
+- Added support for the `[[omp::assume]]` attribute.
```

Comment:
> This should probably go under an OpenMP-specific section as it relates more to OpenMP than anything else.

Should I add an omp-specific section to the Clang release notes? Because there doesn’t seem to be one, and the OpenMP release notes in the `openmp` directory are specifically meant for changes to the runtime—at least according to a comment there.

Reply:
I'd recommend adding a section to clang's release notes for OpenMP in that case.